include/igraph_community.h

```diff
@@ -107,7 +107,7 @@ IGRAPH_EXPORT igraph_error_t igraph_community_edge_betweenness(const igraph_t *g
                                                     igraph_matrix_t *merges,
                                                     igraph_vector_t *bridges,
```

Comment:
`merges` has to become an `igraph_matrix_int_t` in the future; `bridges` needs to be an `igraph_vector_int_t`; again, it's not necessary to do them now, just let's be aware of it.